covigator/__init__.py

```diff
@@ -9,3 +9,18 @@
 CONSERVATIVE_INFRAME_DELETION = "conservative_inframe_deletion"
 DISRUPTIVE_INFRAME_INSERTION = "disruptive_inframe_insertion"
 CONSERVATIVE_INFRAME_INSERTION = "conservative_inframe_insertion"
+INTERGENIC_VARIANT = "intergenic_variant"
+INTERGENIC_DELETION = "intergenic_deletion"
+FRAMESHIFT_VARIANT = "frameshift_variant"
+
+CANONICAL_PROTEIN_MAPPING = {
+            "S": ["S", "s", "spike"],
+            "N": ["N", "n"],
+            "M": ["M", "m"],
+            "E": ["E", "e"],
+            "ORF1ab": ["ORF1ab", "1ab", "orf1ab", "ORF1a", "ORF1b", "orf1a", "orf1b", "nsp3", "nsp6", "nsp5", "nsp4",
+                       "nsp15", "nsp13", "nsp7", "nsp12", "NSP2"],
+            "ORF3a": ["ORF3a", "orf3a"],
+            "ORF8": ["ORF8", "8"],
```

Comment:
In the same spirit as the previous comment, in order to make it more robust add all casings for ORFs, eg: `ORF8` and `orf8`.  Also add all genes even if not present as of today. I see ORF10, ORF7a, ORF7b, ORF3a, but please check.

See https://www.ncbi.nlm.nih.gov/projects/sviewer/?id=NC_045512&tracks=[key:sequence_track,name:Sequence,display_name:Sequence,id:STD649220238,annots:Sequence,ShowLabel:false,ColorGaps:false,shown:true,order:1][key:gene_model_track,name:Genes,display_name:Genes,id:STD3194982005,annots:Unnamed,Options:ShowAllButGenes,CDSProductFeats:true,NtRuler:true,AaRuler:true,HighlightMode:2,ShowLabel:true,shown:true,order:9]&v=1:29903&c=null&select=null&slim=0